ServerApp/config.py

```diff
@@ -0,0 +1,31 @@
+# Statement for enabling the development environment
+DEBUG = True
+HOST = '0.0.0.0'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible binding to all interfaces.](https://app.codacy.com/manual/rejahrehim/SecureTea-Project/pullRequest?prid=5693944)